ruby/vampires.rb

```diff
@@ -0,0 +1,70 @@
+
+applicant_name = #{name}
+  puts "what is your name"
+  name = gets.chomp
+
+def age_correct
+  puts "what is your age"
+  age = gets.chomp.to_i
+  puts "what year were you born"
+  birth_year = gets.chomp.to_i
+if 2017-birth_year == age
+  age_correct = TRUE
+  puts "birth year approved"
+else
+  age_correct = FALSE
+  puts "birth year incorrect"
+
+end
+
+end
+age_correct
+
+def garlic_OK
+likes_garlic = #{garlic_preference}
+  puts "Would you like garlic bread? y/n"
+  garlic_preference = gets.chomp
```

Comment:
Nice work on attempting to encapsulate this data into a method.  One thing that we do want to look out for in methods is that we should be choosing to pass our data in as a parameter.

Running `gets.chomp` inside of a method is not a good practice.  Instead set this method up to evaluate input through a passed argument.